bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/config/DaikinConfiguration.java

```diff
@@ -16,12 +16,18 @@
  * Holds configuration data for a Daikin air conditioning unit.
  *
  * @author Tim Waterhouse - Initial contribution
+ * @author Jimmy Tanagra - Add secure, uuid
  *
  */
 public class DaikinConfiguration {
     public static final String HOST = "host";
+    public static final String SECURE = "secure";
+    public static final String UUID = "uuid";
+    public static final String KEY = "key";
 
     public String host;
-
+    public Boolean secure;
+    public String uuid;
+    public String key;
```

Comment:
Are these new configs backwards compatible with existing setups?

Reply:
Yes they are. Existing config especially those using .things file that don't specify these configs will continue to work. The new configs are not required, at least for the existing adapters. They do need to be provided for the new adapter to work, otherwise a CONFIGURATION_ERROR will be issued on thing status.